Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+#!groovy
```

Comment:
@andrewkroh Have a look at this file. I could see this also becoming useful for us for the `beats` build matrix and stages. Like this we would have more directly the repo itself.